test/PageController-test.js

```diff
@@ -0,0 +1,252 @@
+const chai = require('chai');
```

Comment:
I am pretty much all caught up on all the things on my side, besides small design elements. So I am happy to focus on these tests with you and get them done.

I also will have about 5 tests to add to userrepo-test.js lets bring this thing home.

Reply:
🙌